src/utils/date/format.ts

```diff
@@ -0,0 +1,13 @@
+import { default as fnsFormat } from "date-fns/format";
```

Comment:
copy this function from their github, installing a whole library just to use 1 is a waste